src/Extension.php

```diff
@@ -42,6 +43,7 @@ class Extension implements ExtensionInterface
     */
     const SILVERSTRIPE_ID = 'silverstripe_extension';
 
+    private ?bool $shouldBootstrap = null;
```

Comment:
```suggestion
```
Don't bother caching result, this will hard be called at all

Reply:
Done